services/svc_picasaweb.py

```diff
@@ -105,6 +105,9 @@ def getUrlFromImages(self, types, width, images):
       proposed = images['feed']['entry'][index]['content']['src']
       if self.memorySeen(proposed):
         continue
+      if not self.isCorrectOrientation(images[index]['mediaMetadata'], displaySize):
```

Comment:
This should be after `self.memoryRemember(proposed)` since it will allow other parts of the code to be aware of the skipped image (granted, nothing does this today, but at least it's prepared)

Reply:
good point, I changed it!